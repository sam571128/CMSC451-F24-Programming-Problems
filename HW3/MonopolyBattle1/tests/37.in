54 2916 1000
50 15
36 4
18 2
29 54
44 8
12 11
2 1
37 1
31 25
38 15
9 32
38 10
18 40
28 27
35 40
28 51
2 8
37 14
30 30
54 24
45 42
13 11
34 53
28 39
14 17
30 52
44 53
47 14
33 3
15 52
20 18
37 45
4 46
9 40
17 20
16 34
31 23
16 4
7 31
14 43
45 49
42 34
33 26
43 17
50 50
11 10
3 51
51 16
52 17
32 18
18 21
50 46
29 29
9 47
52 28
9 25
11 25
41 15
4 10
53 44
31 24
31 50
26 10
20 21
5 17
29 32
31 49
23 43
11 30
42 10
43 16
19 41
52 50
5 12
41 23
34 2
40 30
32 9
44 27
21 15
24 28
1 31
7 10
3 17
36 42
46 44
1 30
10 43
18 41
20 42
12 47
12 46
34 22
15 13
33 47
8 22
15 7
26 49
1 4
46 42
42 47
30 6
46 9
49 30
20 24
44 20
49 39
7 3
30 42
14 36
39 25
44 19
54 8
17 46
39 38
38 22
29 41
18 31
37 32
54 17
34 44
11 7
1 40
27 23
42 8
48 15
1 19
11 53
43 28
47 36
39 16
49 1
7 11
47 21
7 50
7 39
5 31
51 3
14 54
47 47
42 33
12 27
39 13
1 39
27 41
41 2
4 23
8 46
2 23
53 5
2 29
50 42
49 52
31 21
43 37
11 16
30 45
41 31
30 38
10 7
46 51
50 41
37 31
29 36
51 1
51 34
39 11
32 25
24 50
29 39
8 51
1 25
38 18
47 9
46 53
26 13
41 44
14 20
4 40
18 48
27 27
48 54
37 43
48 49
18 20
23 41
38 9
42 2
34 37
36 10
6 22
44 38
17 13
20 46
25 39
22 16
38 46
43 44
47 37
30 13
4 12
10 36
6 40
27 34
48 36
37 51
31 42
4 30
20 9
52 20
5 21
20 26
39 6
22 53
47 18
32 41
52 43
13 43
17 33
37 37
15 44
10 44
30 25
41 19
41 28
51 39
10 26
17 52
39 42
44 15
49 6
37 22
47 25
20 44
35 1
50 31
20 11
24 14
33 29
47 20
31 8
43 31
9 41
45 27
2 16
32 8
12 38
22 48
9 48
17 23
27 11
29 6
43 53
1 21
29 37
30 41
39 48
2 12
10 1
21 33
32 10
10 22
36 30
42 9
3 25
36 18
49 37
8 18
23 49
1 37
29 42
15 45
47 46
33 54
21 29
23 24
25 52
23 11
32 4
32 22
22 54
18 18
51 11
33 49
50 2
45 24
1 50
30 3
20 49
46 46
16 9
5 23
15 10
51 23
46 32
23 19
50 53
21 45
36 8
50 38
24 47
23 16
38 41
46 6
40 22
16 15
8 9
41 39
31 3
23 52
20 52
21 6
48 11
21 37
1 48
54 42
34 40
4 29
37 46
4 49
4 42
37 30
52 38
2 48
31 7
41 48
7 27
29 31
50 28
32 26
12 18
22 15
9 24
32 15
6 39
29 26
36 48
35 52
8 37
3 49
38 6
8 45
48 23
42 53
7 7
27 25
42 31
24 44
46 43
47 8
31 22
44 1
33 32
26 28
36 44
18 11
52 54
6 18
40 11
8 12
48 52
49 46
32 14
17 35
45 45
38 42
24 37
39 39
29 21
26 2
44 10
20 45
1 35
51 30
19 18
14 6
23 27
17 47
42 5
49 22
16 8
52 24
21 26
37 44
5 34
45 17
14 4
27 14
43 35
28 41
3 22
25 50
51 25
23 30
12 26
9 5
30 22
20 8
32 2
17 32
42 37
38 47
3 16
30 5
15 30
48 6
35 12
12 19
8 24
50 9
26 35
35 22
11 50
41 33
9 37
33 24
11 20
27 54
19 24
34 41
39 27
26 8
27 21
30 15
23 36
54 31
7 46
7 44
21 2
31 20
47 32
44 23
14 25
44 11
1 13
16 31
7 52
22 11
9 39
12 4
24 35
6 21
54 29
24 19
11 26
9 51
23 32
33 19
14 3
13 23
29 30
20 53
53 30
9 36
53 35
24 36
4 19
29 43
43 13
12 7
26 26
37 17
52 46
15 35
21 30
33 39
21 10
51 37
17 3
15 29
38 2
16 39
31 17
21 7
16 22
8 50
4 32
14 18
15 15
26 47
12 2
25 41
19 36
26 29
33 40
52 41
38 13
27 44
35 48
53 42
30 35
45 2
5 16
42 25
22 18
39 4
33 5
16 23
4 39
43 48
29 1
9 9
45 37
15 53
16 24
32 27
32 34
28 20
10 29
18 36
1 41
40 12
27 49
52 3
6 37
3 30
45 52
35 35
47 51
37 29
16 13
39 51
39 29
21 22
23 3
19 40
48 5
2 41
27 12
11 37
35 46
6 20
44 5
49 27
14 9
2 22
10 14
22 19
18 27
41 14
10 5
23 37
47 52
48 44
25 12
29 22
35 5
24 42
38 19
3 23
9 22
36 12
52 45
22 13
14 24
32 24
41 21
20 14
21 27
41 22
53 45
34 26
21 24
42 35
29 2
20 15
51 18
46 22
53 27
7 9
35 25
54 45
45 22
12 20
14 5
47 28
8 1
36 11
2 9
43 43
47 11
32 35
3 15
16 33
29 34
26 37
40 1
6 47
2 52
21 25
50 22
5 32
33 36
13 21
11 28
48 10
34 24
30 50
23 6
10 32
37 36
33 34
31 18
32 31
49 50
44 28
41 27
5 7
12 3
51 40
43 21
25 16
53 47
36 33
31 48
54 2
23 20
8 47
34 34
25 8
2 39
5 53
19 50
1 5
19 25
41 45
37 2
34 42
44 43
1 38
19 47
23 12
51 36
37 54
6 35
46 34
7 41
44 44
2 26
7 33
3 38
19 35
37 6
10 28
15 17
20 5
11 34
40 25
4 2
40 3
17 24
54 21
44 40
1 36
41 10
5 26
13 20
48 20
7 16
14 45
27 42
51 43
8 21
36 14
46 13
23 33
8 42
3 42
44 31
20 37
21 35
25 9
51 10
54 22
10 52
53 3
11 46
25 22
6 5
30 4
20 1
2 2
18 33
11 49
16 41
35 23
51 13
2 33
32 29
46 18
27 38
21 17
6 52
17 44
14 39
51 14
28 37
31 12
34 21
33 6
13 29
44 16
50 1
51 54
38 4
4 37
39 30
14 51
37 47
2 44
52 7
33 25
14 44
48 42
52 11
11 19
19 30
48 14
27 50
53 36
38 1
16 43
13 4
23 44
13 35
47 54
10 46
6 15
7 36
22 41
4 6
7 22
18 47
30 36
11 27
35 49
27 20
2 38
23 54
50 20
16 52
42 52
53 48
53 41
28 4
36 25
13 53
47 39
28 19
14 15
10 38
36 22
28 44
36 51
9 45
41 17
5 48
34 11
16 18
34 43
15 40
42 50
12 14
30 33
16 1
4 43
2 21
26 18
4 35
8 16
8 52
7 47
14 38
31 37
41 3
34 36
42 20
24 46
53 40
35 44
54 6
44 24
25 34
1 28
44 9
13 24
7 42
28 45
24 33
9 14
40 5
8 29
35 31
16 51
27 16
10 17
25 40
52 21
8 7
49 25
46 25
37 12
6 29
24 49
44 48
11 21
31 39
42 14
13 13
12 35
48 8
1 33
7 53
30 9
14 31
39 9
27 5
35 17
4 52
15 32
32 42
21 32
36 41
6 45
4 1
29 9
32 38
30 14
45 18
7 18
28 10
11 31
33 16
5 37
7 21
43 12
49 9
52 25
24 5
24 21
18 53
14 11
15 39
17 50
29 16
32 11
8 49
15 38
11 54
8 32
23 45
9 2
36 2
54 1
1 47
53 33
8 43
14 40
21 38
43 10
1 1
34 38
14 47
31 16
7 15
23 21
42 16
13 46
6 10
46 41
19 42
44 34
11 6
32 44
13 54
8 34
1 14
25 14
34 50
47 29
45 13
34 54
22 1
3 35
54 4
21 34
9 27
43 26
43 19
33 31
54 12
26 15
27 3
11 35
16 10
19 39
44 13
32 33
2 15
16 26
3 21
18 51
35 30
47 13
51 2
7 34
48 37
52 42
41 49
39 54
48 43
19 6
42 26
23 7
25 53
50 17
4 36
42 42
12 36
5 2
38 5
7 6
51 4
13 42
51 12
42 24
45 26
4 21
39 49
30 31
50 47
40 35
10 9
26 42
20 32
43 8
3 3
42 32
25 36
38 12
25 48
50 43
18 19
37 33
12 37
31 6
12 50
7 13
24 24
45 53
33 23
32 39
30 47
18 38
52 23
52 37
37 38
14 7
46 33
11 48
17 21
22 3
51 32
22 12
44 45
13 44
53 15
10 16
39 20
44 52
33 22
26 33
45 51
15 19
14 35
11 9
48 38
54 7
54 50
49 11
31 54
38 51
40 7
37 53
31 29
18 22
21 51
54 19
49 49
39 22
26 38
24 32
44 29
18 17
43 4
7 4
53 8
41 8
7 5
9 13
51 24
33 1
34 35
48 46
14 28
18 29
34 20
3 28
35 27
52 34
42 45
5 25
30 34
25 33
51 44
23 35
41 25
49 23
11 5
45 21
48 25
49 36
23 15
26 40
22 45
10 19
39 31
28 34
3 48
49 44
34 45
53 21
15 42
54 53
34 52
39 23
41 16
47 42
10 4
49 47
11 41
37 3
5 15
54 3
25 29
51 33
10 50
26 6
1 26
12 21
36 39
27 18
20 25
43 18
29 49
37 40
9 16
42 1
21 49
14 8
31 4
7 43
21 21
47 41
15 27
31 38
44 30
48 13
21 23
10 51
53 46
43 36
49 41
38 24
14 2
46 28
49 51
54 33
5 22
32 52
48 2
4 31
34 5
8 31
22 9
38 39
28 17
28 18
3 13
12 51
36 45
49 17
53 10
53 17
3 9
53 50
1 44
45 50
34 16
46 12
31 26
48 39
44 47
19 33
14 12
35 13
50 10
13 28
24 34
48 16
11 4
44 18
30 16
1 3
21 43
9 3
3 26
16 44
46 16
45 32
20 35
19 48
44 21
36 13
6 13
19 20
34 19
47 27
5 38
17 15
43 5
36 15
19 10
17 6
19 45
13 31
51 46
24 8
34 4
49 16
50 26
23 31
4 9
13 22
49 31
17 37
52 8
30 8
24 18
29 46
28 15
43 39
22 47
42 19
34 31
33 27
21 16
32 3
43 29
53 11
38 45
53 49
52 26
49 53
3 7
18 34
33 9
41 37
37 41
47 48
6 31
54 18
4 26
35 3
32 1
41 40
41 13
18 3
43 27
54 35
43 40
39 7
5 13
22 49
27 10
35 43
26 11
47 24
17 4
12 54
5 24
6 9
23 34
3 18
46 29
18 7
8 48
43 41
42 6
7 37
15 28
15 46
15 14
4 5
49 13
17 45
33 48
52 12
14 52
43 6
47 4
36 16
48 32
46 14
17 25
25 13
21 44
16 48
6 50
5 6
36 54
32 53
2 42
5 11
5 49
13 30
11 45
4 27
43 9
54 32
22 2
44 54
36 7
31 28
14 33
19 23
28 35
19 37
51 42
12 6
53 9
47 2
20 13
44 50
44 39
50 35
11 18
6 30
18 43
34 25
52 31
40 13
51 7
2 3
30 2
4 41
30 29
33 2
44 49
14 10
52 33
5 4
42 28
15 11
9 19
7 38
36 23
41 35
17 9
46 26
27 29
31 2
29 7
54 16
8 6
31 27
25 43
48 19
2 54
8 35
25 38
46 47
7 40
43 51
7 20
45 28
46 8
16 16
24 11
1 18
50 5
4 47
43 47
19 54
7 23
20 29
38 3
39 32
5 45
1 27
12 15
8 23
8 2
8 40
18 13
14 16
15 49
27 28
45 15
28 22
22 32
38 54
10 10
31 43
34 49
9 43
38 38
32 21
6 7
30 17
38 33
18 32
27 2
38 32
32 5
9 21
46 3
5 27
3 6
1 42
49 43
46 7
54 46
36 52
1 29
48 33
50 48
2 36
44 37
45 1
33 51
40 36
26 36
52 39
19 13
22 7
40 47
48 9
11 15
18 23
39 14
22 23
1 2
44 46
3 32
38 16
22 17
39 26
52 9
47 19
29 17
2 10
28 1
42 40
23 26
7 29
17 27
25 21
2 28
18 8
29 3
49 19
25 17
33 50
12 32
31 46
29 19
50 45
48 29
6 25
22 5
46 1
15 3
7 45
24 3
43 15
44 14
27 33
19 2
14 29
6 32
22 40
45 7
42 41
15 2
9 50
30 19
3 39
38 48
10 6
53 26
25 32
9 15
20 48
41 46
2 18
10 30
50 8
5 52
33 12
27 9
13 18
28 32
38 29
34 6
35 42
14 30
48 45
20 22
25 26
20 30
3 31
48 18
3 14
36 5
6 46
5 51
17 8
45 47
35 36
12 34
13 2
27 4
28 36
33 7
31 40
50 23
15 4
3 53
3 5
6 14
49 40
48 12
28 53
20 28
21 31
16 36
52 5
27 30
45 25
14 37
28 49
40 8
14 14
5 5
22 33
13 32
11 39
33 4
40 41
16 30
16 35
43 14
34 23
24 48
49 32
48 21
28 29
11 2
33 44
8 5
25 27
35 28
5 33
12 33
5 43
30 26
42 22
12 12
16 3
23 8
8 27
25 4
9 11
50 13
2 27
18 49
8 11
17 36
37 52
24 43
46 4
36 38
26 31
8 53
50 24
18 30
24 40
44 12
11 23
22 37
9 7
45 29
2 43
41 30
23 25
27 36
38 26
27 26
54 54
14 49
31 35
6 51
12 48
16 14
54 39
21 50
12 23
49 21
21 11
53 32
2 53
42 7
45 19
50 51
3 2
47 49
4 13
6 3
52 18
31 30
10 20
18 45
31 52
15 54
41 54
11 1
51 21
20 31
25 10
37 23
38 37
43 20
45 16
18 28
17 17
27 15
47 1
8 8
4 25
14 23
5 40
54 37
51 35
12 13
49 48
18 42
26 45
46 20
42 18
10 31
23 2
7 28
40 50
22 36
47 34
22 51
29 4
54 44
37 20
28 6
9 1
5 10
28 14
32 32
26 20
13 50
43 54
27 6
39 15
34 46
40 26
18 9
53 51
5 35
32 37
13 17
17 40
15 6
10 49
49 10
14 53
29 11
36 37
16 12
2 37
45 39
20 43
23 1
27 31
11 43
20 12
7 24
31 53
30 7
45 8
21 18
46 10
27 7
17 30
51 51
8 33
50 44
1 12
41 51
15 22
1 10
2 45
4 34
6 1
48 27
40 24
46 30
45 23
13 38
50 49
17 18
7 48
6 38
6 8
53 6
37 10
45 40
2 40
22 42
42 43
26 27
39 40
6 34
24 26
9 44
23 17
34 33
54 9
48 48
12 10
48 34
45 3
44 42
29 14
22 8
52 15
2 34
8 38
46 37
13 49
35 14
35 15
4 4
15 18
24 16
33 10
18 39
6 19
54 40
36 40
27 39
16 21
37 5
12 30
33 37
35 19
32 46
35 18
16 50
52 16
28 38
10 8
24 51
22 35
29 5
5 30
50 39
39 5
27 32
28 3
8 4
17 31
22 26
27 52
12 41
40 42
3 33
46 11
42 15
21 9
48 7
27 17
43 22
23 5
52 49
48 31
16 7
40 23
23 46
52 47
13 12
4 14
6 23
39 18
23 40
48 4
21 46
11 40
13 27
32 48
30 18
30 48
12 42
39 1
11 44
9 10
46 40
12 25
35 32
50 54
21 40
39 8
4 48
38 40
47 10
39 10
6 16
12 52
51 26
16 40
54 30
29 13
36 29
33 15
6 17
9 4
47 45
29 52
48 28
25 19
19 12
39 37
7 8
38 7
45 12
11 8
24 2
40 2
22 43
39 44
26 17
5 19
8 26
47 33
45 43
37 19
46 38
53 1
16 38
2 13
51 5
8 13
33 13
41 5
9 33
36 36
4 38
51 27
53 34
39 43
21 19
24 17
54 48
46 45
10 54
45 30
29 48
40 33
52 51
17 1
54 11
36 28
26 41
34 3
53 29
18 35
25 42
49 42
33 18
52 14
10 24
31 47
53 25
9 54
39 41
47 22
36 47
45 41
37 49
31 10
24 20
12 1
45 4
36 19
22 6
21 14
14 22
41 1
51 38
35 11
33 21
35 39
1 9
35 21
19 1
53 19
9 26
39 36
18 4
48 50
10 2
16 11
26 19
3 54
49 45
37 25
11 12
7 49
27 19
12 43
33 17
32 36
52 6
18 26
35 6
41 32
43 11
40 53
18 52
29 25
50 37
8 36
39 17
46 50
1 6
23 18
39 19
24 7
4 53
25 3
43 30
45 35
31 41
26 5
30 53
13 7
51 15
23 48
26 25
30 51
20 23
19 52
30 1
49 26
5 46
53 7
27 24
24 1
50 40
34 51
32 50
22 31
6 54
10 42
26 52
49 20
50 33
44 35
33 46
54 26
35 16
33 42
51 17
24 9
53 13
36 31
12 49
15 8
52 10
42 29
33 45
39 52
19 43
2 32
45 54
26 24
28 7
38 35
7 2
6 28
39 47
15 47
19 7
5 50
11 14
30 10
3 45
29 38
52 32
1 8
49 18
29 28
29 33
25 49
34 12
16 46
17 26
30 37
1 52
34 1
19 3
54 5
51 8
8 17
26 22
3 40
13 26
33 43
53 24
48 17
39 45
46 49
40 38
25 28
9 34
30 12
13 33
46 17
4 3
20 51
18 5
24 15
35 9
49 14
11 51
21 3
50 7
16 20
5 20
3 36
1 43
45 48
49 35
9 31
31 15
25 46
45 5
1 15
42 21
20 54
9 8
42 38
12 5
45 31
23 38
2 46
11 32
1 7
34 47
11 52
51 19
9 12
10 12
16 42
47 31
7 19
52 2
20 7
21 41
32 20
35 26
13 6
3 10
51 53
7 14
15 1
33 20
47 30
35 33
4 18
43 52
13 37
45 38
20 41
53 23
25 37
10 39
17 38
22 24
25 31
11 17
35 8
49 3
31 9
40 34
19 53
13 52
25 1
26 48
45 44
22 28
6 2
40 28
33 28
28 42
36 35
16 47
15 12
12 16
12 44
3 11
8 3
47 53
1 45
42 30
54 14
35 29
4 11
26 51
39 3
50 3
40 45
43 25
47 17
11 24
24 41
31 31
25 6
2 31
35 54
6 24
36 26
36 32
16 37
16 5
27 22
40 15
41 41
34 9
52 1
30 49
12 9
25 7
2 51
17 19
28 26
6 43
41 12
18 1
30 28
13 15
40 19
37 4
19 27
36 46
20 10
11 36
10 27
3 43
17 12
28 2
53 14
40 52
24 53
9 28
16 53
11 33
35 50
3 46
51 28
16 45
34 18
1 22
52 13
24 22
35 53
21 36
32 51
39 35
18 44
37 9
2 6
41 42
51 22
34 17
48 22
42 12
35 47
35 45
26 1
18 14
54 23
41 36
13 10
47 3
27 45
3 47
14 13
27 37
38 53
29 20
25 25
40 29
26 30
5 9
15 23
17 11
29 53
13 25
48 41
35 4
41 52
38 28
43 34
19 17
39 24
38 8
15 34
19 38
13 34
19 21
48 53
30 39
32 17
20 50
32 47
8 15
4 28
17 41
17 5
31 32
52 22
53 18
30 24
37 42
12 22
50 11
18 15
10 21
23 53
9 49
3 20
17 54
28 31
6 53
7 54
42 3
52 4
22 4
36 17
14 26
34 15
6 41
21 1
47 5
17 22
32 54
10 15
9 46
11 42
23 23
28 52
43 42
47 16
2 24
38 21
21 47
51 29
35 38
37 48
13 45
27 46
16 49
52 30
15 9
42 44
24 39
49 33
3 34
22 34
33 53
29 8
53 16
13 40
46 23
28 28
6 11
2 5
36 9
35 2
1 23
31 45
39 33
25 20
5 39
29 44
38 14
22 46
46 24
15 31
4 51
48 24
32 43
19 32
45 14
25 54
44 25
54 27
51 52
13 19
35 51
9 17
46 27
24 27
22 38
45 46
20 38
23 28
6 12
32 12
41 11
46 2
7 51
2 30
15 48
26 21
22 14
47 44
34 48
6 6
37 28
10 53
38 17
38 49
5 29
29 45
18 54
44 17
28 24
21 52
22 29
34 10
24 10
41 53
9 20
13 8
29 50
10 34
29 35
47 23
30 11
38 31
20 39
20 47
43 46
24 29
39 34
23 10
2 11
23 42
47 6
5 42
12 28
37 18
44 41
40 54
42 4
25 2
3 8
15 51
31 11
54 25
1 32
28 48
35 37
19 26
30 23
24 30
30 40
17 34
51 41
40 44
11 3
22 10
44 51
43 24
19 16
53 20
38 43
44 6
34 27
30 44
9 30
45 6
48 1
53 37
33 33
19 51
2 49
25 35
54 43
23 47
41 7
40 39
40 20
15 21
10 41
32 28
31 51
40 27
21 53
39 28
15 20
4 33
36 34
14 19
27 8
24 6
35 34
49 24
9 53
24 38
41 18
19 46
41 29
6 26
44 32
29 10
40 31
44 33
40 4
36 3
27 35
18 12
5 41
18 6
54 41
23 14
20 16
54 20
21 20
5 14
47 7
27 1
45 36
6 27
48 40
41 20
7 26
17 10
28 47
48 47
42 27
54 10
19 31
35 10
37 26
20 2
15 16
23 29
29 40
54 38
4 45
41 47
13 48
48 30
19 28
2 17
14 42
32 13
17 53
25 51
32 40
28 46
53 31
32 49
9 18
6 49
53 43
28 30
4 54
54 49
32 23
49 15
27 48
11 11
23 9
17 2
11 22
28 13
29 18
50 30
38 52
44 7
22 25
52 48
51 31
5 47
26 53
49 5
19 11
42 46
28 40
22 39
33 14
1 16
18 16
41 43
3 50
22 30
51 20
28 9
33 30
32 6
3 37
50 21
53 53
27 43
37 15
48 51
12 31
27 51
54 52
52 44
17 42
25 30
26 46
7 35
23 39
19 44
37 27
7 17
15 26
3 41
7 12
6 48
4 24
51 49
12 8
13 5
30 32
13 36
20 27
2 47
20 36
10 48
4 20
20 40
47 35
8 41
13 14
26 32
13 3
25 45
17 43
50 34
15 36
13 41
38 30
47 12
24 25
24 13
16 54
46 15
21 54
42 13
46 48
8 44
31 36
37 8
49 29
29 23
40 16
38 23
5 44
40 18
54 34
52 27
26 43
50 18
46 54
19 49
13 16
42 49
54 13
52 53
47 43
44 3
28 8
37 39
51 6
43 33
27 40
26 9
1 20
14 21
26 4
10 35
18 24
42 23
45 34
21 28
46 35
3 24
45 10
14 41
4 44
5 28
51 9
46 5
45 11
22 22
41 34
39 21
17 16
5 54
33 35
31 13
1 11
20 4
46 19
16 2
17 14
49 2
12 24
53 22
7 32
25 44
32 16
46 39
54 15
14 34
19 9
19 4
38 36
19 22
10 23
2 7
32 30
38 27
48 3
39 2
40 32
20 6
30 54
16 19
46 31
29 24
34 14
28 11
36 49
9 6
47 50
23 4
44 36
42 17
19 15
47 26
31 1
27 13
18 46
6 42
4 15
41 6
21 8
40 10
23 22
40 40
10 18
30 46
51 48
42 39
49 28
8 39
13 9
51 50
49 34
28 5
44 2
12 53
38 25
24 45
42 36
40 21
35 41
40 37
36 50
2 4
3 12
25 24
50 19
1 54
37 7
23 51
38 11
32 45
40 14
5 18
31 14
36 21
50 14
30 43
41 38
16 25
8 19
28 54
43 3
41 4
5 36
50 36
41 50
18 37
34 32
30 21
42 51
22 50
43 32
38 34
14 46
43 7
17 39
8 30
15 41
47 15
53 54
32 7
3 52
46 21
37 24
17 7
15 50
15 43
49 4
48 26
28 16
50 4
25 11
25 5
36 27
24 31
12 45
11 13
17 49
31 5
28 50
51 47
34 28
23 50
38 20
52 19
41 24
26 7
31 19
51 45
26 39
26 44
28 43
52 36
37 35
24 54
50 29
21 5
44 22
19 5
17 29
49 38
54 28
13 39
10 13
50 32
10 37
40 48
4 50
3 4
1 34
9 52
14 50
41 9
33 38
24 4
20 34
38 44
47 38
49 54
1 53
28 25
45 33
11 47
29 51
11 29
37 11
50 6
37 13
36 43
7 1
54 47
20 20
4 16
21 12
25 15
22 52
14 27
20 3
34 13
54 51
12 17
5 1
6 33
26 50
39 46
43 45
36 20
33 8
23 13
19 8
40 6
29 27
3 19
9 29
50 52
33 11
16 29
13 51
52 35
26 3
10 25
16 17
54 36
31 33
17 51
14 48
19 19
2 14
34 39
2 50
18 10
11 38
9 23
49 12
19 29
36 1
10 40
13 47
14 1
28 23
1 24
2 25
6 4
22 44
12 40
49 7
42 54
10 45
6 36
53 4
21 42
25 18
24 12
52 29
8 14
40 46
40 17
18 50
15 33
53 39
27 47
33 52
4 7
39 12
46 36
34 8
9 42
25 47
35 7
43 2
10 47
20 19
40 43
26 34
26 14
39 53
12 39
1 51
43 1
9 35
53 12
5 3
15 25
44 4
38 50
50 27
48 35
24 52
43 50
40 9
22 27
27 53
10 11
10 33
29 15
1 17
2 20
44 26
5 8
37 21
53 38
36 6
15 37
28 21
37 34
16 32
4 17
43 23
15 24
52 52
3 1
33 41
13 1
28 12
42 11
45 20
50 25
26 54
50 16
34 7
30 27
8 25
22 20
8 20
35 24
17 48
43 38
47 40
19 14
42 48
9 38
7 25
28 33
43 49
35 20
8 10
25 23
34 29
22 21
21 4
40 49
37 16
29 47
50 12
29 12
21 39
2 19
19 34
32 19
52 40
16 6
26 23
3 27
20 17
1 46
39 50
20 33
36 53
12 29
31 34
24 23
3 44
21 13
18 25
4 8
7 30
45 9
49 8
30 20
15 5
8 54
31 44
53 52
26 12
46 52
14 32
2 35
10 3
36 24
53 28
8 28
4 22
53 2
16 28
21 48
16 27
1 49
40 51
37 50
26 16
34 30
17 28
41 26
3 29
6 44
