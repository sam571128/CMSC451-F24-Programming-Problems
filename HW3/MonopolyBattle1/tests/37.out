3185 3185 3185 3185 3185 3185 3185 3185 3185 3185 3185 3185 3185 3185 3185 3185 3185 3185 3185 3479 3185 3185 3185 3185 3185 3185 3185 3185 3185 4106 3185 3185 3185 4518 3185 3185 3185 3185 3185 3185 3185 3185 3185 3185 3185 3185 4270 3185 3185 3185 3185 3185 3185 3185 